Remove payloads flatpak and rpmostree which are not supported by oe.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Rebase for anaconda 34.

Signed-off-by: Kai Kang <kai.kang@windriver.com>

Rebase for anaconda 37.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/anaconda.py | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/pyanaconda/anaconda.py b/pyanaconda/anaconda.py
index ba93a098ac..a4be1b5d16 100644
--- a/pyanaconda/anaconda.py
+++ b/pyanaconda/anaconda.py
@@ -81,10 +81,7 @@ class Anaconda(object):
             # Get the type of the DBus payload module if any.
             payload_type = self._get_dbus_payload_type()
 
-            if payload_type == PAYLOAD_TYPE_RPM_OSTREE:
-                from pyanaconda.payload.rpmostreepayload import RPMOSTreePayload
-                payload = RPMOSTreePayload()
-            elif self.opts.liveinst:
+            if self.opts.liveinst:
                 from pyanaconda.payload.live import LiveOSPayload
                 payload = LiveOSPayload()
             elif payload_type == PAYLOAD_TYPE_LIVE_IMAGE:
